src/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs

```diff
@@ -503,6 +503,7 @@ public void Select_LargeList_Throws_ArgumentOutOfRange()
             Assert.Throws<ArgumentOutOfRangeException>(() => Socket.Select(null, null, largeList, -1));
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Bug in AcceptAsync that dereferences null SAEA argument")]
```

Comment:
We need an open GitHub issue to track investigating/fixing tests/product code.  You don't need to open an different issue for each test you disable. You can do what I've done already and have an issue for each library.  Please then list that issue number in the SkipOnTargetFramework attribute.